Find a functionally equivalent code:android.media.MediaPlayer.TimeProvider.onPaused:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -4,13 +4,16 @@
             Log.d(TAG, "onPaused: " + paused);
         if (mStopped) {
             // handle as seek if we were stopped
+            mStopped = false;
+            mSeeking = true;
             scheduleNotification(NOTIFY_SEEK, 0);
         } else {
             // special handling if player disappeared
             mPausing = paused;
+            mSeeking = false;
             scheduleNotification(REFRESH_AND_NOTIFY_TIME, 0);
         }
     }

